api_tests/lib/create_actors.ts

```diff
@@ -1,14 +1,25 @@
 import { Actors } from "./actors";
 import { Actor } from "./actors/actor";
 import { createActor } from "./create_actor";
+import { HarnessGlobal, mkdirAsync, rimrafAsync } from "./utils";
+import path from "path";
+
+declare var global: HarnessGlobal;
 
 export async function createActors(
-    logFileName: string,
+    testName: string,
     actorNames: string[]
 ): Promise<Actors> {
     const actorsMap = new Map<string, Actor>();
+    const testFolderName = path.join(
+        global.logRoot,
+        "/tests/",
```

Comment:
```suggestion
        "tests",
```
The point of using `path.join` is to not assume that `/` or `\` are used on the target system.